### PR TITLE
Add some scenarios

### PR DESCRIPTION
**Pull request by chinua**
Adding samples for few scenarios
-Change task hierarchy
-Create records
-Update records
-CUD records
-Insert task

### CODE REVIEW

File: schedule-api/C#/SDK-Sample/Program.cs [hunk 1]
```diff
@@ -1,103 +1,93 @@
...
+        {
+            var sleepTime = 5000 + (100 * expectedNumber);
+            int numberOfRecords = -1;
+            int maxRetry = 10;
+            int retryCount = 0;
+            while (expectedNumber != numberOfRecords && retryCount++ != maxRetry)
+            {
+                Thread.Sleep(sleepTime);
+                numberOfRecords = countRecordsFunction();
+            }
```
Comment (boctorino) [inline schedule-api/C#/SDK-Sample/Program.cs:88]: spacing

**Review by boctorino — APPROVED** [review]

**Comment by boctorino** [comment]
update the readme file to reflect your changes.

File: schedule-api/C#/SDK-Sample/Program.cs (hunk 1 above)
Comment (chinua) [reply schedule-api/C#/SDK-Sample/Program.cs:88]: We can fix that later

**Comment by chinua** [comment]
Adding samples for few scenarios
-Change task hierarchy
-Create records
-Update records
-CUD records
-Insert task

**Comment by chinua** [comment]
Re-open

**Pull request merged**